Riot/Modules/KeyVerification/Common/KeyVerificationCoordinator.swift

```diff
@@ -162,6 +165,12 @@ final class KeyVerificationCoordinator: KeyVerificationCoordinatorType {
     }
     
     private func didCancel() {
+        if self.completeSecurityCoordinator != nil && childCoordinators.count > 1 {
+            NSLog("[KeyVerificationCoordinator] didCancel: popToRootModule");
+            self.navigationRouter.popToRootModule(animated: true)
+            return
```

Comment:
It would be more readable to use an `else` below and removing this `return`.

Reply:
well, it is like a `guard  let`